### PR TITLE
PyUp configuration

### PR DESCRIPTION
**Pull request by benjaoming**
### Summary

Do not pin dev and build dependencies that are re-downloaded by bots at every build

### Reviewer guidance

Will close all PRs that are related to dev/test/build dependencies.

Please review now, the bot is coming back in 2 hours!!!

### References

n/a

---

### Contributor Checklist

- [ ] PR has the correct target milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, it has been assigned or requests review from someone and labeled as 'needs review'
- [ ] PR has been fully tested manually
- [ ] Documentation is updated
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Link to diff of internal dependency change is included
- [ ] Screenshots of any front-end changes are in the PR description
- [ ] PR does not introduce [accessibility regressions](http://kolibri.readthedocs.io/en/develop/dev/manual_testing.html#accessibility-a11y-testing)
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] You've added yourself to AUTHORS.rst if you're not there

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] PR has been fully tested manually

### CODE REVIEW

File: .pyup.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# update schedule
+# default: empty
+# allowed: "every day", "every week", ..
+schedule: "every day"
```
Comment (indirectlylit) [inline .pyup.yml:4]: maybe once a week? :)

**Comment by jamalex** [comment]
I have only one opinion on this, and defer to @rtibbles and @indirectlylit on the rest: Would it be possible to have it bundle all upgrades into a single PR (less overhead, and single test run), rather than exploding into separate PRs for each package?

File: .pyup.yml (hunk 1 above)
Comment (benjaoming) [reply .pyup.yml:4]: I think what we're seeing now is the "problem" of not integrating upstream upgrades in due time, we can maybe talk about it at the dev meeting, but I think @pyup-bot should just go nuts and hit us with these updates every day so we don't linger. We can add deliberate filters as I'm doing in this PR for the things we have chosen not to update.

**Comment by benjaoming** [comment]
@jamalex 

> Would it be possible to have it bundle all upgrades into a single PR (less overhead, and single test run), rather than exploding into separate PRs for each package?

When reviewing, reading change logs, and reading test output, it's a lot easier to have the one-update-per-PR model.

I think its monolith version is also a technical challenge for PyUp... knowing what's already an open PR and whats not.

**Comment by benjaoming** [comment]
Argh, sorry, but the bot was already back.. just wanted to get this in before another 20 PRs that I just closed get opened up...

**Comment by benjaoming** [comment]
Please continue reviewing :)

**Pull request merged**